docs/src/examples/components/Avatar/Variations/AvatarExampleImageCustomization.shorthand.tsx

```diff
@@ -12,21 +12,6 @@ const AvatarExampleImageCustomizationShorthand = () => (
       image="public/images/avatar/large/jerry.png"
       status={{ color: 'green', icon: 'check', title: 'Available' }}
     />
-    &emsp;
-    <Avatar
-      image={
-        // This example does not react to the avatar size variable
-        // and otherwise produces bad results when border is applied compared to "normal" image
-        <Icon
-          name="lock"
-          circular
-          bordered
-          variables={{ color: 'blue' }}
-          styles={{ boxSizing: 'border-box', padding: '8px' }}
-        />
-      }
-      status={{ color: 'green', icon: 'check', title: 'Available' }}
-    />
```

Comment:
Our examples should not share tricks as use cases to avoid confusions.

If we want to support icons on `Avatar` it should be an `icon` slot? 🤔

Reply:
This is already used on the client's side, so would suggest proposing fix for this, rather then just remove it...

Reply:
Not sure about this, even previously it had custom `styles` what is definitely not the best option

Reply:
Restored this one as we decided during offline discussion.